src/schemas/javascript/angular.intellisense.js

```diff
@@ -241,26 +241,26 @@
             * @typedef {Object} httpConfig
             * @property {String} method HTTP method (e.g. 'GET', 'POST', etc)
             *    @property {String} url Absolute or relative URL of the resource that is being requested.
-            * @property {Object.<string|Object>} params  � Map of strings or objects which will be turned
+            * @property {Object.<string|Object>} params  � Map of strings or objects which will be turned
```

Comment:
I don't know what this symbol is/was. My editor changed this automatically :thinking: